api/fasit.go

```diff
@@ -548,10 +549,14 @@ func (fasit FasitClient) mapToNaisResource(fasitResource FasitResource, property
 		resource.certificates = files
 
 	} else if fasitResource.ResourceType == "applicationproperties" {
+		lineFilter, err := regexp.Compile(`[\p{L}\d_.]+=[\p{L}\d_,]+`)
+		if err != nil {
+			return NaisResource{}, fmt.Errorf("unable to compile regex: %s", err)
+		}
+
 		for _, line := range strings.Split(fasitResource.Properties["applicationProperties"], "\n") {
 			line = strings.TrimSpace(line)
-
-			if len(line) > 0 {
+			if lineFilter.MatchString(line) {
```

Comment:
should we log non matching strings?

Reply:
Sure, would make it easier to debug when someone complains.

Reply:
But, maybe avoid logging empty strings. So I'm doing a check for length.